.github/ci_path_filters.yaml

```diff
@@ -42,3 +42,49 @@ material_components:
  - 'ui-libraries/material/**'
  - *files_in_root
  - *ci_config
+
+internal: &internal
+ - 'internal/**'
+ - 'helper_crates/**'
+ - *files_in_root
+ - *ci_config
+
+api_cpp:
+ - 'api/cpp/**'
+ - 'tools/compiler/**'
+
+api_python:
+ - 'api/python/**'
+
+api_node:
+ - 'api/node/**'
+
+api_rs:
+ - 'api/rs/**'
+
+tests:
+ - 'tests/**'
+
+examples:
+ - 'examples/**'
+ - 'demos/**'
+
+vsce:
+ - *files_in_root
+ - *ci_config
+ - *internal
+ - 'api/rs/**'
+ - 'editors/vscode/**'
+ - 'tools/lsp/**'
```

Comment:
Should we add `docs/common/src/utils/slint.tmLanguage.json` here?

Reply:
Good point.

There is editors/vscode/slint.tmLanguage.json

I wonder if it would run the CI on changes there.

Reply:
Indeed, the symlink is not enough to get a detection. As shown in my experiment in  https://github.com/slint-ui/slint/pull/9829